docs/client/Android/_options.mdx

```diff
@@ -36,6 +36,9 @@ You can pass in an optional parameter `options` in addition to `sdkKey` and `use
   - This can mean that a different set of fields will map to the same cached user.  The intent is to improve cache hit rate, and provide better defaults
   - If you have critical fields that define a user differently which are not considered IDs, you can override the cache key we generate using this function
 
+- **evaluationCallback**: `((config: BaseConfig) -> Unit)`, v4.24.0+
+  - A function that can be supplied to the sdk that will be called whenever a gate, config, experiment or layer is checked. The function takes a BaseConfig as the only parameter which is the result of the check made.
```

Comment:
how do I filter down to the entity that was checked? What is the baseconfig type?  this is missing some info